website/docs/r/search_index.html.markdown

```diff
@@ -254,7 +254,7 @@ An [Atlas Search analyzer](https://docs.atlas.mongodb.com/reference/atlas-search
         * `type` - Must be `uaxUrlEmail`
         *  `max_token_length` - The maximum number of characters in one token.
 
-* `tokenFilters` - Array containing zero or more token filters. Always require a type field, and some take additional options as well:
+* `token_filters` - Array containing zero or more token filters. Always require a type field, and some take additional options as well:
```

Comment:
The other two corrects are correct but I don't think this one is.  tokenFilters is correct (https://www.mongodb.com/docs/atlas/atlas-search/analyzers/custom/) - because it is simply included in the content one would pass into `analyzers`.  This is likely left over documentation from when we had a more formal design of the search resource.  I'll create an internal ticket to fix this part as the tests use the same and I'm not sure it's correct.  Please go ahead and remove this from the PR and we'll investigate further.

Reply:
Got you! I think then there are a few more cases of JSON filed names vs Terraform attributes confusions on the list, let me quickly fix that.

Reply:
https://help-hr.mongodb.com/hc/en-us/requests/53603

Reply:
@daniel-orlov this PR looks really good. The only line still off is 257, the one this comment is on.  All the others are fixed and look great. Could you push a fix for this and we'll merge so this will be in our next release?

Reply:
@themantissa Thank you for your comments! I am afraid I have to disagree with you again - it should be either token_filters in snake_case here AND on line 259 or it will just be misleading. I understand that camelCase is used in the Mongo API and you provided the reference, but here we are documenting terraform objects' fields, it makes no sense to confuse the user by using the wrong attribute name.  

Moreover the docs have been stating It the same [here](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/website/docs/guides/1.0.1-upgrade-guide.html.markdown?plain=1#L40) and [here](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/website/docs/guides/1.0.1-upgrade-guide.html.markdown?plain=1#L73) as well.

I once again kindly ask you to reread the references and ask yourself a question - what are we referencing here - a Mongo API field or a Terraform attribute? 

Because if you are right and it should be tokenFilters indeed, there's no way other corrections I made are valid  - because the Mongo API docs state otherwise (e.g. char_filters).

If you have time, I am open to scheduling a Google Meet (or other conferencing tool of your choice) session and we can go over it again.

Reply:
@daniel-orlov I'm glad you disagreed because I was yet again looking at the tests and not the actual resource code.  You are correct - this is the right casing/variable!  Glad to have some help on this one :) I'll resolve this comment and merge the PR as is (and get that test fixed too by our devs!)